pkg/synopsysctl/cmd_deploy.go

```diff
@@ -314,6 +331,22 @@ var deployCmd = &cobra.Command{
 			return err
 		}
 
+		// Create default sizes
+		_, err = sizeClient.SynopsysV1().Sizes(operatorNamespace).Create(size.GetSize("small"))
```

Comment:
Can we create []string{"small", "medium", "large"} and iterate them to create the CR?

Reply:
done